src/config.ts

```diff
@@ -7,19 +8,31 @@ import { previousState } from './shared';
 
 let configPath: string | undefined;
 
-export const findConfig = async (config?: string) => {
+export const writeToSettings = async (settings: CSpellSettings | { cspell: CSpellSettings }) => {
+	await writeFile(configPath!, JSON.stringify(settings, null, 4));
+};
+
+export const findOrCreateConfig = async (config?: string) => {
 	// Try to locate a config file in the current working directory, or with the `config` option if it was provided.
 	const configSource = config ?? (await searchForConfig(process.cwd()))?.__importRef?.filename;
 
+	// LocalConfigPath is a path to a config file in the user's configuration directory (platform dependent).
+	const localConfigPath = (await import('application-config-path')).default('cspell.json');
```

Comment:
Separating this into a different variable defeats the point of the dynamic import. Importing packages has seemed to massively decrease performance (see #2), so dynamically importing would have prevented the need to import another package if the config was already found. If you still think a separate variable is a good idea, you should import `application-config-path` regularly.

Reply:
This is the last chunk I'm changing for the reviews you made, but just thinking here, I need to check if the final path is from default config path so I can check if the file exists, and, if it doesn't, create it before returning the path to the main CLI flow.

I got the deal with the dynamic import on a single variable but I have to check if the path is from the configuration directory at all to do the subsequent checkings. According to Bundlephobia, [`application-config-path` is smaller than 1kB minified](https://bundlephobia.com/package/application-config-path@1.0.0), I'll do some performance testing for this import with and without dynamic strategy to see how it affects running performance before deciding this!

Reply:
The test was superficial because was running the mocks on it and writing operations will afect final time but by changing importing strategies these are the numbers
|              | await import | regular import |
|--------------|--------------|----------------|
| avg/100 runs | 12ms         | 4.6ms          |

Reply:
Alright, cool, you can just change this to a regular import then.

Reply:
I like findDefaultConfigPath 🙂

Reply:
Done!